### PR TITLE
Fix SendQueue not setting lastReceiverResponse when recv handshake ack

### PR DESCRIPTION
**Pull request by huffman**

### CODE REVIEW

**Comment by huffman** [comment]
This is a small networking change that fixes an issue when it takes > 5 seconds to receive a handshake ack. This makes sure that once you receive that ack you don't immediately timeout and drop your SendQueue.

To test:
 1. Go to your local sandbox with a fresh cache. Everything should load. Try doing this 3 times.
 1. Go to Welcome. Make sure you can see other avatars.
 1. Go back to your local sandbox. Everything should load.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/184/HighFidelity-Beta-PR10552-f0c3a0ac2876c984900572d7123f89cdb985347e.exe

File: libraries/networking/src/udt/SendQueue.cpp [hunk 1]
```diff
@@ -241,6 +241,9 @@ void SendQueue::handshakeACK(SequenceNumber initialSequenceNumber) {
             std::lock_guard<std::mutex> locker { _handshakeMutex };
             _hasReceivedHandshakeACK = true;
         }
+
+        _lastReceiverResponse = QDateTime::currentMSecsSinceEpoch();
```
Comment (ZappoMan) [inline libraries/networking/src/udt/SendQueue.cpp]: any chance we could use a `std::` function for this?

Comment (huffman) [inline libraries/networking/src/udt/SendQueue.cpp]: We could, but we use currentMSecsSinceEpoch in other areas of the code. I can make that change if we think it is important, but I would prefer doing it as a separate PR.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/184/HighFidelity-Beta-PR10552-f0c3a0ac2876c984900572d7123f89cdb985347e.dmg

**Comment by Nex-Pro** [comment]
Checking this

File: libraries/networking/src/udt/SendQueue.cpp (hunk 1 above)
Comment (ZappoMan) [inline libraries/networking/src/udt/SendQueue.cpp]: ok - but yes... where ever possible I'd like us to move toward stl instead of qt in cases like this

**Comment by Nex-Pro** [comment]
@Huffman seems all ok.   
Apart from loading at welcome was very slow, took a couple of minutes to see the other avatars. Don't think it is down to this PR. 
Lemme know if you want logs.

**Comment by huffman** [comment]
Thanks @Nex-Pro!

File: libraries/networking/src/udt/SendQueue.cpp (hunk 1 above)
Comment (huffman) [inline libraries/networking/src/udt/SendQueue.cpp]: Agreed

**Pull request merged**